CHANGELOG.md

```diff
@@ -4,6 +4,44 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.3](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.3) - 2021-12-21
+
+Zebra's latest beta works towards enforcing all consensus rules by validating JoinSplit Groth16 proofs
+used by Sprout transactions. We have also added security and network improvements, and have also
+added some metrics to help diagnose networking issues.
+
+### Added
+
+#### Consensus
+
+- Validate JoinSplit proofs (#3128, #3180)
+
+#### Networking
+
+- Add and use `debug_skip_parameter_preload` config option in tests (#3197)
+- Disconnect from outdated peers on network upgrade (#3108)
+
+#### Metrics
+
+- Add diagnostics for peer set hangs (#3203)
+- Add debug-level Zebra network message tracing (#3170)
+
+### Fixed
+
+- Stop ignoring some connection errors that could make the peer set hang (#3200)
+- Spawn initial handshakes in a separate task, Credit: Equilibrium (#3189)
+- Fix coinbase height deserialization to reject non-minimal encodings (#3129)
+- Stop doing thousands of time checks each time we connect to a peer  (#3106)
+- Revert "Stop ignoring some completed Responses" (#3274)
```

Comment:
Since this reverts part of PR #3200, which is also in this release, we don't need to document it.

```suggestion
```